commit

```diff
@@ -29,11 +39,11 @@ for branch in "${IGNORED_BRANCHES[@]}";
 	fi
 done
 
+
 if [ "$IS_IGNORED" == false ]
 then
-    echo "Commiting with branch name -> $CURRENT_BRANCH"
-    eval $COMMIT_WITH_BRANCH
+    commitWithBranch $1
 else
-    echo "Commiting with default branch"
-    eval $DEFAULT_COMMIT
+    colorEcho "GREEN" "Commit translation: \"$1\""
```

Comment:
why not move this to a function as well if we're doing it for everything else?

Reply:
I don’t think function is needed now. What do you think about colorEcho and function?

Reply:
Maybe you should add screenshots to the PR. How it looks now. It's a good idea to get visual feedback with colors on how the commit message has been written

Reply:
On the third line of the output you already know if `smart-commit` was useful or not. Thus, any verbosity is not required. If it was for debugging or anything else, it would have made sense. Keep it simple, it should be fine!

Reply:
The coloring is not required in my opinion. What you can do is add an additional flag to show these messages. By default, it should show none.